azbus/msgsender.go

```diff
@@ -2,13 +2,19 @@ package azbus
 
 import (
 	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus"
 )
 
 type MsgSender interface {
 	Open() error
 	Close(context.Context)
 
 	Send(context.Context, *OutMessage) error
+	NewMessageBatch(context.Context) (*OutMessageBatch, error)
```

Comment:
Not sure if I like those additions to this interface, can't we have standalone type to deal with message batching and only have the `sendbatch` here? or perhaps we just want a new interface for batch sender altogether?

Reply:
We could but that is a lot more work. I would be happy to raise a TD ticket to further improve this. My focus was getting us to a place where we can make appropriate use of message batching at all